src/main/java/com/scottlogic/sorts/KeywordSearch.java

```diff
@@ -0,0 +1,55 @@
+package com.scottlogic.sorts;
+
+import com.scottlogic.UserPost;
+import com.scottlogic.filters.ContentFilter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KeywordSearch {
+
+    public List<UserPost> keywordSearchSort(List<UserPost> inputList, String keyWord) {
```

Comment:
Cool, I have a couple of tweaks to suggest, but they are hard to put into text, so let's chat about them when we talk after lunch!